### PR TITLE
ci(semaphore.yml): fix styling and add CI style check

### PR DESCRIPTION
**Pull request by ArturBa**
Fix styling for existing component, fix commands for stylefix, add command for style check, add
stylecheck to CI

#767

### CODE REVIEW

File: .semaphore/semaphore.yml [hunk 1]
```diff
@@ -73,6 +73,18 @@ blocks:
             - checkout
             - cache restore node-$(checksum yarn.lock)
             - yarn eslint workspaces/*/cypress
+  - name: Typescript Code Style Check
+    dependencies:
+      - Eslint
+    task:
+      jobs:
+        - name: typecodestype
```
Comment (kantord) [inline .semaphore/semaphore.yml:81]: A small typo here maybe? I think you meant sth like "typecodestyle"? I would maybe just put "types" there

**Comment by kantord** [comment]
Almost perfect, just please fix the comment about the typo :-) :rocket:

File: .semaphore/semaphore.yml (hunk 1 above)
Comment (ArturBa) [reply .semaphore/semaphore.yml]: fixed

**Comment by kantord** [comment]
:100: 

Thank you!!!!!

**Pull request merged**